acados/ocp_nlp/ocp_nlp_sqp_rti.c

```diff
@@ -516,10 +511,6 @@ void ocp_nlp_sqp_rti_preparation_step(void *config_, void *dims_,
     mem->time_lin = 0.0;
     mem->time_reg = 0.0;
 
-    int N = dims->N;
-
-    int ii;
-
 #if defined(ACADOS_WITH_OPENMP)
```

Comment:
better move also the beginning and end of parallel region in the new routine to alias memory, otherwise there is the future risk of calling it from somewhere else without them

Reply:
To be more precise, the issue is that the `nowait` in the `omp for` means that there is not an explicit barrier after the parallelized for loops, and the implicit barrier is at the end of the parallel region.
So either the beginning and end of parallel regions are also moved inside the new routine, or at least an explicit barrier is added at the end of the new routine.

Reply:
Same for ocp_nlp_sqp.c.

Reply:
I addressed this in https://github.com/acados/acados/pull/772/commits/d66ce703d708eaaa88b8943b11844a987b1c272d
Playing a bit with openmp, I found an issue, but this is definitely not introduced in the current PR:  https://github.com/acados/acados/issues/782